selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -106,6 +106,14 @@ public ProjectExplorer(
     String JAVA_SOURCE_FOLDER = "javaSourceFolder";
   }
 
+  public interface ProjectExplorerOptionsMenuItem {
+    String MAXIMAZE = "contextMenu/Maximize";
+    String MINIMAZE = "contextMenu/Minimize";
+    String COLLAPSE_ALL = "contextMenu/Collapse All";
+    String REFRESH_MAIN = "contextMenu/Refresh 'main'";
+    String LINK_WITH_EDITOR = "contextMenu/Link with editor";
+  }
+
```

Comment:
Can we replace pubic interface on public static constans in the place-? And syntax err.: MAXIMAZE-> MAXIMIZE, MINIMAZE -> MINIMIZE

Reply:
+1